doc/classes/TileSet.xml

```diff
@@ -464,6 +478,14 @@
 				Proxied tiles can be automatically replaced in TileMap nodes using the editor.
 			</description>
 		</method>
+		<method name="set_custom_data_name">
+			<return type="void" />
+			<argument index="0" name="layer_index" type="int" />
+			<argument index="1" name="layer_name" type="String" />
+			<description>
+				set the name of a given custom data layer identified by it's index. Names are identifiers of the layer therefore if the name is already taken it will fail and raise an error.
```

Comment:
```suggestion
				Sets the name of the custom data layer identified by the given index. Names are identifiers of the layer therefore if the name is already taken it will fail and raise an error.
```